tests/Models/EmployeeTest.php

```
@@ -161,9 +161,114 @@ public function testHasGrantReturnsTrueForActualGrant(): void
+        $employeeData = [
+            'username'      => 'granttester',
+            'password'      => password_hash('password123', PASSWORD_DEFAULT),
+            'hash_version'  => 2,
+            'language_code' => 'en',
+            'language'      => 'english'
+        ];
+
+        $employeeModel = model(Employee::class);
+        $employeeModel->save_employee($personData, $employeeData, $grantsData, NEW_ENTRY);
+
+        return $personData['person_id'];
```

PHPStan (2.2.7)

[error] Offset 'person_id' does not exist on array{first_name: 'Grant', last_name: 'Tester', email: 'granttester@test.com', phone_number: '555-5678'}.


(offsetAccess.notFound)